data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
             updatedBy: "Alexander", updatedAt: "2020-02-03",
         },
     },
+    {
+        meta: {
+            name: "12/06/2020 Olympia",
+            link: "https://www.startnext.com/12062020",
+            keywords: ["petition", "networking", "event", "grassroots"],
+            label: "12-06-2020-olympia",
+        },
+        description: {
+            content: "One-time event where up to 90,000+ people come together to rent the Olympia stadium Berlin to sign various petitions."
+            + " In Germany, petitions must be discussed by parliament if they get more than 50,000 signatures. In addition there will be talks and presentations on currently pressing social and environmental problems.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2019](https://www.startnext.com/12062020)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "German Zero",
+            link: "https://germanzero.de/",
```

Comment:
link to english summary: https://germanzero.de/english_summary
We should discus whether we would like to adress this to a german audience or a english speaking audience. My optinion is to put the english link. Readers from germany still can go to the main web page and english speakers can get an overview of the organization